src/components/material-grid-related/MaterialGridRelated.tsx

```diff
@@ -19,26 +39,123 @@ const MaterialGridRelated: React.FC<MaterialGridRelatedProps> = ({ work }) => {
   const title = t("materialGridRelatedTitleText");
 
   const pid = getWorkPid(work);
-  const { data } = useWorkRecommendationsQuery({
-    pid: pid,
-    limit: 8
-  });
 
-  if (!data?.recommend.result) {
+  const { creators } = work.manifestations.bestRepresentation;
+  const { series } = work;
+  const seriesObject = first(series);
+  const flattenedCreators = flattenCreators(creators);
+  const creatorCqlString = prepareCreatorCql(flattenedCreators);
+
+  const { data: recommendationData, isLoading: recommendationLoading } =
+    useWorkRecommendationsQuery({
+      pid,
+      limit: 8
+    });
+
+  const { data: creatorData, isLoading: creatorLoading } =
+    useComplexSearchWithPaginationQuery({
+      cql: creatorCqlString,
+      limit: 8,
+      offset: 0,
+      filters: {}
+    });
+
+  const { data: seriesData, isLoading: seriesLoading } =
+    useComplexSearchWithPaginationQuery({
+      cql: `term.series='${seriesObject?.title}'`,
```

Comment:
Consider whether we should disable (`enabled: false`) this hook if there is no series for the object.

It seems to me like we would get a `term.series=''` query but that does not make much sense to me.